docs/resources/filesystem.md.erb

```diff
@@ -25,12 +25,15 @@ A `filesystem` resource block declares tests for disk space in a partition:
 
     describe filesystem('/') do
       its('size') { should be >= 32000 }
+      its('free') { should be >= 5000000 }
+      its('type') { should cmp 'ext4' }
     end
 
 where
 
 * `filesystem('/')` states that the resource will look at the root (/) partition.
-* `size` is measured in kilobytes (KB).
+* `size` is the total partition size and is measured in kilobytes (KB).
+* `free` is the available space on the partition and is measured in kilobytes (KB).
```

Comment:
Based on further discussion, I believe that we cannot in fact know the units here for either `size` or `free`: the units may vary based on OS, disk size, fs type, and block size.

Based on that, I think releasing `free` at this time is not appropriate.  It would not be safe to do a comparison test (Imagine asking someone if you had $20 in your wallet, and you responded, "Well, I have 3 bills.").

We could however add `free_percent`, since while we don't know the units, we do know they will be the same for both `free` and `size`.